__tests__/app.test.js

```diff
@@ -98,12 +114,17 @@ describe('error handling', () => {
     });
     test('returns a 400 if review_id is invalid', () => {
         return request(app).get('/api/reviews/banana').expect(400).then((response) => {
-            expect(response._body.msg).toBe('review id invalid')
+            expect(response._body.msg).toBe('invalid input')
         })
     });
     test('returns a 404 if review_id is out of range', () => {
         return request(app).get('/api/reviews/999').expect(404).then((response) => {
             expect(response._body.msg).toBe('review not found')
         })
     });
+    test('returns a 400 if passed an invalid username', () => {
+        return request(app).post('/api/reviews/WalterWhite/comments').expect(400).then((response) => {
+            expect(response._body.msg).toBe('invalid input')
+        })
+    });
```

Comment:
Great so far but it's short a few tests. You'll need:-
- A 201 test to make sure that unnecessary properties on the request object are ignored
- A test for an a non-existent id e.g. 9001
- A 400 for missing properties on the request e.g no username / body